doc/source/ray-observability/ray-logging.rst

```diff
@@ -2,7 +2,30 @@
 
 Logging
 =======
-This document will explain Ray's logging system and its best practices.
+This document explains Ray's logging system and related best practices.
+
+Internal Ray Logging Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When ``import ray`` is executed, Ray's logger is initialized, generating a sensible configuration given in ``python/ray/log.py``. The default logging level is ``logging.INFO``.
+
+All ray loggers are automatically configured in ``ray._private.ray_logging``. To change the ray library logging config:
+
+.. code-block:: python
+
+   import logging
+
+   logger = logging.getLogger("ray")
+   logger # Modify the ray logging config
+
+Similarly, to modify the logging configuration for any ray subcomponent, specify the appropriate logger name. For example, to modify the `ray.rllib` log configuration,
+
+.. code-block:: python
+
+   import logging
+   logger = logging.getLogger("ray.rllib")
```

Comment:
Can you add examples for all libraries (data, tune, rllib, air, train, workflow)?

Reply:
Yep, I added a section showing how to grab handles for each of the ray loggers, and how to set the logging level and add a handler for two different ray subpackages.